index.html

```diff
@@ -63,5 +63,176 @@ <h6><a href="#">LET’S CONNECT</a></h6>
+                <li class="year"><img src="images/mobile/work/Counter.png" alt="image" class="counter-2"/><h6>2015</h6></li>
+              </ul>
+            </div>
+          </div>
+
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul>
+              <li class="tag-1"><h6>html</h6></li>
+              <li class="tag-2"><h6>css</h6></li>
+              <li class="tag-3"><h6>javaScript</h6></li>
+            </ul>
+          <div class="action">
+            <button type="button" class="button-text">See Project</button>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section id="about">
+    <div class="about-content">
+      <div class="about-top">
+        <h2>About Myself</h2>
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need
+           coded, don’t hestiate to contact me.
+        </p>
+        <div class="social-media">
+          <div class="let-talk-text">
+            <h6><a href="#">LET’S CONNECT</a></h6>
+
+            <ul>
+              <li> <a href="#"><img src="images/mobile/twitter.png" alt="image"/></a></li>
+              <li> <a href="#"><img src="images/mobile/linkedin.png" alt="image"/></a></li>
+              <li> <a href="#"><img src="images/mobile/github.png" alt="image"/></a></li>
+              <li> <a href="#"><img src="images/mobile/m.png" alt="image"/></a></li>
+              <li> <a href="#"></a> <img src="images/mobile/fingers.png" alt="image"/></li>
+           </ul>
+          </div>   
+        </div>
+        <div class="action">
+          <button type="button" class="button-text">See my resume</button>
+        </div>
+      </div>
+      <div class="frame-12">
+        <div class="frame-7">
+          <h6>Langguages</h6>
+          <button class="frame-7-button" type="button"><i class="fas fa-caret-down"></i></button>
+        </div>
+        <div class="list-language">
+          <div class="list-language-1">
+            <img src="images/mobile/work/js.png" alt="image"/>
+            JavaSript
+          </div>
+
+          <div class="list-language-2">
+            <img src="images/mobile/work/html.png" alt="image"/>
+            HTML
+          </div>
+
+          <div class="list-language-3">
+            <img src="images/mobile/work/css.png" alt="image"/>
+            CSS
+          </div>
+        </div>
+
+        <div class="frame-8">
+          Frameworks
+          <i class="fas fa-arrow-right"></i>
+        </div>
+        <img src="images/mobile/work/separator-bottom.png" class="separator-1" alt="images"/>
+
+        <div class="frame-9">
+          Skills
+          <i class="fas fa-arrow-right"></i>
+        </div>
+        <img src="images/mobile/work/separator-bottom.png" class="separator-2" alt="images"/>
+      </div>
+    </div>
+  </section>
```

Comment:
- [ ] Going by the images below your design looks quite different from what is defined in the Figma template. Please do well to use make your design is as similar to the template as needed.
- [ ] Your **ABOUT MYSELF** section is aligned somewhat to the middle unlike your **WORKS** section
- [ ] There should be some margin at the top of the **ABOUT MYSELF** section and a right border-radius styling
- [ ]  Your 'LET'S CONNECT' text should be bigger and aligned to the left
- [ ] Your button text, text color and size should be similar to what is on the Figma template.
- [ ] Your JavaScript, HTML and CSS tag styling are aligned to the right. Please, align them to the left as in the template
- [ ] The Frameworks and Skills parts should be aligned to the left just like the Languages. They should also be made bolder
- [ ] For the right arrows and down arrow, you can visit this [Boxicons](https://boxicons.com/?query=chevron) search to see some useful icons

This is your design:

![ogah_about_new](https://user-images.githubusercontent.com/36542986/133866885-8a416d3d-d517-4972-b8e2-7102a17cb3b9.jpg)

But the Figma template looks like this:

![ogah_about](https://user-images.githubusercontent.com/36542986/133866896-63c57d94-1b33-4e97-b74b-ee944a172258.jpg)

Reply:
Thank you for mention

Reply:
There was a page overflow that causes it